frankenphp.go

```diff
@@ -501,6 +501,80 @@ func ServeHTTP(responseWriter http.ResponseWriter, request *http.Request) error
 	return nil
 }
 
+//export go_putenv
+func go_putenv(str *C.char, length C.int) C.bool {
+	// Create a byte slice from C string with a specified length
+	s := C.GoBytes(unsafe.Pointer(str), length)
+
+	// Convert byte slice to string
+	envString := string(s)
+
+	// Check if '=' is present in the string
+	if idx := strings.IndexByte(envString, '='); idx != -1 && idx < len(envString)-1 {
```

Comment:
Sorry for nitpicking but in the `FOO=` case, shouldn't the variable be set to an empty string instead of being deleted?

Reply:
I'll double check the behavior here.

Reply:
Don't worry about nitpicking. Making sure we get this right is very important.